build.gradle.kts

```diff
@@ -19,6 +19,7 @@ buildscript {
 
 plugins {
   id("com.diffplug.spotless") version versions.spotless
+  id("dev.zacsweers.kgp-150-leak-patcher") version versions.kgpLeakPatcher
```

Comment:
Can we add a comment on the `Kotlin` dependency version declaration to remind anyone who changes that version to check if this plugin will still be required for that version?

Reply:
Good point.